packages/jam/jam-host-calls/accumulate/bless.ts

```diff
@@ -49,14 +51,15 @@ export class Bless implements HostCallHandler {
     // `m`: manager service (can change privileged services)
     const manager = getServiceId(regs.get(IN_OUT_REG));
     // `a`: manages authorization queue
-    // NOTE It can be either ServiceId (pre GP 067) or memory index (GP ^067)
     const authorization = regs.get(8);
     // `v`: manages validator keys
-    const validator = getServiceId(regs.get(9));
+    const delegator = getServiceId(regs.get(9));
+    // `r`: manages creation of new services with id within protected range
+    const registrar = Compatibility.isGreaterOrEqual(GpVersion.V0_7_1) ? getServiceId(regs.get(10)) : tryAsServiceId(0);
```

Comment:
```suggestion
    const registrar = Compatibility.isGreaterOrEqual(GpVersion.V0_7_1) ? getServiceId(regs.get(10)) : tryAsServiceId(2**32 - 1);
```
I'd suggest using some large value instead of `0` to easier detect bugs (if they happen).